tensorflow/core/kernels/fft_ops.cc

```diff
@@ -27,24 +27,11 @@ limitations under the License.
 #include "tensorflow/core/platform/types.h"
 #include "tensorflow/core/util/work_sharder.h"
 
-#if GOOGLE_CUDA
-#include "tensorflow/core/platform/stream_executor.h"
-
 namespace tensorflow {
 
-namespace {
-// TODO(vrv/zhifengc): Refactor AsDeviceMemory() into GPUUtil.
-template <typename T>
-perftools::gputools::DeviceMemory<T> AsDeviceMemory(const T* cuda_memory) {
-  perftools::gputools::DeviceMemoryBase wrapped(const_cast<T*>(cuda_memory));
-  perftools::gputools::DeviceMemory<T> typed(wrapped);
-  return typed;
-}
-}  // end namespace
-
-class FFTGPUBase : public OpKernel {
+class FFTBase : public OpKernel {
```

Comment:
Sorry for nitpicking, can you go through the comments here and add periods? Our style guide prefers punctuation:
https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar